### DISCUSSION TITLE
Recommendations: where to buy

### DISCUSSION THREAD

**Discussion by dietervansteenwegen** [mathoudebine/turing-smart-screen-python#388] [Q&A] [answered]
Hi,

I'm looking to buy a compatible 3.5" screen. However, since there seem to be several different versions that look similar but are incompatible, I'm wondering where to buy. I live in Europe. 

Aliexpress doesn't show many hits when searching for "Turing 3.5" or "XuanFang 3.5". At least not with the same manufacturer name in the actual description. Amazon has some, but not sure how to be sure they are really compatible. The manufacturer itself has a website but does not seem to sell from that website.

Where do people buy their screens?

Thanks!

dieter

**Comment by mathoudebine** [comment, marked as answer]
Hi @dietervansteenwegen!
I usually buy mine on Aliexpress but I agree they are hard to find because the brand is never in the ad...
Usually searching for `smart screen 3.5"` or `secondary screen 3.5"` is enough as most products are the same.

To know the screen model you have to check the ad text for the software link or screen capture e.g. if the software is the `35inchENG.rar` package then it's a Turing 3.5 screen
Check [Vendor Apps](https://github.com/mathoudebine/turing-smart-screen-python/wiki/Vendor-apps) for details
If no software is mentioned in the ad, my advice is don't buy the product because you cannot be sure of the model.

Since you live in Europe as I do, here are some links I used to buy my previous screens:
Turing 3.5"
https://aliexpress.com/item/1005005641370235.html
https://aliexpress.com/item/1005003723773653.html (black or white version available)

Xuanfang rev. b
https://aliexpress.com/item/1005005647817383.html

UsbPcMonitor 3.5"
https://aliexpress.com/item/1005003931363455.html

It is very hard to find XuanFang 3.5" flaghisp (with RGB LEDs), it seems all have been sold.
Compare the total price inc. shipping for your address, shipping delay and seller reputation to make your choice.

Good shopping! 🛒

**Comment by dietervansteenwegen** [comment]
Cool, thanks. Is there any reason to choose the Turing over the Xuafang or UsbPcMonitor (or the other way around)?
They look like clones to me...

**Reply by mathoudebine** [reply]
You're right, all 3.5" models are clones: they all have the same resolution and (slow) refresh rate because they use the same hardware components. Their differences are the functions they may offer with their original software, but if you use this project then all the 3 screens will be supported the same way.
I did not test them enough to know if some models are more stable over time

**Comment by kruuth** [comment]
If you're in the USA, the EDIY is an OG Turing.  It even came in a Turing box.  I got it on Amazon here:
https://www.amazon.com/dp/B09TTHZYSH?ref=ppx_yo2ov_dt_b_product_details&th=1
